CHANGELOG.md

```diff
@@ -6,19 +6,20 @@
 * `DissimilarityMatrix` now has a new constructor method called `from_iterable`. ([#1343](https://github.com/biocore/scikit-bio/issues/1343)).
 * `DissimilarityMatrix` now allows non-hollow matrices. ([#1343](https://github.com/biocore/scikit-bio/issues/1343)).
 * `DistanceMatrix.from_iterable` now accepts a `validate=True` parameter. ([#1343](https://github.com/biocore/scikit-bio/issues/1343)).
+* Added `IntervalMetadata` and `Interval` classes in `skbio.metadata._interval` to allow store, query, and manipulate information of a sub-region of a sequence. ([#1414](https://github.com/biocore/scikit-bio/issues/1414)).
```

Comment:
See my previous comment about removing `_interval` from the path. Don't put private package details in the changelog, it is intended for scikit-bio users.

Also, the current wording doesn't make sense, remove the word `allow`.